examples/misc/tickedstroke_demo.py

```diff
@@ -0,0 +1,101 @@
+"""
+============================
+TickedStroke Patheffect Demo
+============================
+
+Matplotlib's :mod:`.patheffects` can be used to alter the way paths
+are drawn at a low enough level that they can affect almost anything.
+
+The :doc:`patheffects guide</tutorials/advanced/patheffects_guide>`
+details the use of patheffects.
+
+The `~matplotlib.patheffects.TickedStroke` patheffect illustrated here
+draws a path with a ticked style.  The spacing, length, and angle of
+ticks can be controlled.
+
+See also the :doc:`contour demo example
+</gallery/lines_bars_and_markers/lines_with_ticks_demo>`.
+
+See also the :doc:`contours in optimization example
+</gallery/images_contours_and_fields/contours_in_optimization_demo>`.
+"""
+
+import matplotlib.patches as patches
+from matplotlib.path import Path
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as patheffects
+
+# Direct to path
+fig, ax1 = plt.subplots(figsize=(6, 6))
```

Comment:
```suggestion
fig, ax = plt.subplots(figsize=(6, 6))
```
No need to count the axes across different figures. Same for `ax2`, `ax3` below.